src/test/java/StudentTest.java

```diff
@@ -0,0 +1,102 @@
+import com.endava.internship.collections.Student;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.time.Month;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Same here, remove wildcard imports, and move static imports above, right after the package.